accepted/future-releases/nnbd/feature-specification.md

```diff
@@ -1236,18 +1248,92 @@ the `LEGACY_ERASURE` of the types.
 
 ### Const evaluation and canonicalization
 
-With unsound null checking, all generic const constructors and generic
-const literals are treated as if all type arguments passed to them were legacy
-types (both at the top level, and recursively over the structure of the
-types), regardless of whether the constructed class is defined in a legacy
-library or not, and regardless of whether the constructor invocation or
-literal occurs in a legacy library or not.  This ensures that const objects
-continue to canonicalize consistently across legacy and opted-in libraries.
+Const evaluation is modified so that legacy and opted-in instances canonicalize
+more consistently as defined below, using the operation **LEGACY_TYPE**(`T`) on
+types `T` defined as follows.
+
+- **LEGACY_TYPE**(`T`) = `T` if `T` is `dynamic`, `void`, `Null`
+- **LEGACY_TYPE**(`T`) = `T*` if `T` is `Never` or `Object`
+- **LEGACY_TYPE**(`FutureOr<T>`) = `FutureOr<S>*`
```

Comment:
It seems that the meaning of `S` is missing for this case.

Reply:
Fixed.